sums.rb

```diff
@@ -0,0 +1,28 @@
+class Sum1
+    attr_accessor :total
+
+    def initialize(param_1, param_2)
+        @param_1 = param_1
+        @param_2 = param_2
+        @total = param_1 + param_2
+    end
+end
+
+class Sum2
+    def initialize(a, b)
+        # a & b - instance variables
+        @a = a
+        @b = b
+        @total = a + b
+    end
+
+    def new_total
+        sum2_total = @a + @b
```

Comment:
sum2_total is kind of unnecessary in this case. We should just return `@a + @b`

Reply:
@andresochoa91 I removed sum2_total